cms/grading/tasktypes/Batch.py

```diff
@@ -79,14 +79,23 @@ class Batch(TaskType):
     outcome to stdout and the text to stderr.
 
     """
+    # Some local constants used in this class.
+    CORRECT_OUTPUT_FILENAME = "res.txt"
+    CHECKER_FILENAME = "checker"
+    OUTPUT_EVAL_DIFF = "diff"
+    OUTPUT_EVAL_CHECKER = "comparator"
+    COMPILATION_ALONE = "alone"
+    COMPILATION_GRADER = "grader"
```

Comment:
There's two other hardcoded filenames, i.e., `grader.%l.` (the source) and `grader` (the executable). You could consider moving them here as well.